paig-server/backend/requirements.txt

```diff
@@ -35,4 +35,4 @@ fasteners
 APScheduler==3.10.4
 pydantic[email]
 posthog
-paig-authorizer-core==0.0.1
+paig-authorizer-core
```

Comment:
We already had discussion around not keeping any open versions as this could lead to issues 
https://github.com/privacera/paig/pull/96#discussion_r1782936810